tests/test_tools.py

```diff
@@ -12,24 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import argparse
 import socket
 import sys
 import threading
+import unittest
 
 import mock
 from six.moves.urllib import request
-import unittest2
 
 from oauth2client import client
 from oauth2client import tools
 
-try:
-    import argparse
-except ImportError:  # pragma: NO COVER
-    raise unittest2.SkipTest('argparase unavailable.')
```

Comment:
REVIEWER(S): This part is the only non-trivial change, since the `argparse` import will succeed for any version after  (but not including) Python 2.5